ui/Screen/Org/Fund.svelte

```diff
@@ -1,21 +1,15 @@
 <script>
-  import { transactions as store } from "../../src/transaction.ts";
+  import { transactions as txStore } from "../../src/transaction.ts";
+  import { org, orgTransactions } from "../../src/org.ts";
   import { Wallet, Remote } from "../../DesignSystem/Component";
 
   export let params = null;
-
-  const orgTransactions = transactions => {
-    return transactions.filter(tx => {
-      return "domainId" in tx.messages[0]
-        ? tx.messages[0].domainId == params.id
-        : null;
-    });
-  };
+  const accountId = $org.data.accountId;
```

Comment:
```suggestion
  $: accountId = $org.data.accountId;
```

Shouldn't this be reactively assigned to be up to date with the value in the store?

Reply:
Nah this creates an infinite loop cause the component is already contained in a reactive fetch in the `org` component

Reply:
No strings! https://github.com/radicle-dev/radicle-upstream/pull/691/commits/29a7c250a0a1f39cd8246c3e43af52d528730de0
🧨